src/OWML.ModHelper.Events/HarmonyHelper.cs

```diff
@@ -6,6 +6,13 @@
 
 namespace OWML.ModHelper.Events
 {
+	public static class ClearPatch
```

Comment:
can you put this in the bottom or in a new file?

Reply:
I can move it to Patches class actually

Reply:
done